fink_filters/filter_on_axis_grb/filter.py

```diff
@@ -0,0 +1,186 @@
+# Copyright 2019-2022 AstroLab Software
+# Author: Roman Le Montagner
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pyspark.sql.functions import pandas_udf
+from pyspark.sql.types import BooleanType
+
+from scipy import special
+from math import sqrt
+
+from fink_filters.tester import spark_unit_tests
+
+
+def bronze_events(fink_class, realbogus_score):
+    """
+    Return alerts spatially and temporally consistent with a gcn alerts
+    Keep alerts with real bogus score higher than 0.9
+    and the alerts classified as "SN candidates", "Unknown", "Ambiguous"
+
+    Parameters
+    ----------
+    fink_class : pd.Series
+        Fink classification
+    realbogus_score : pd.Series
+
+    Return
+    ------
+    f_bronze : pd.Series
+        alerts falling in the bronze filters
+
+    Examples
+    --------
+    >>> df = pd.read_parquet(grb_output_data)
+    >>> df["f_bronze"] = bronze_events(df["fink_class"], df["rb"])
+    >>> len(df[df["f_bronze"]])
+    25
+    """
+    f_bogus = realbogus_score >= 0.5
+    f_class = fink_class.isin(["SN candidate", "Unknown", "Ambiguous"])
```

Comment:
Here there is a potential problem. If SIMBAD is down, you will reject all alerts. 

Alternatively, you can add a guard against downtime of SIMBAD. You could have another filter which is:

```python
f_fail = fink_class.apply(lambda x: x.startswith('Fail'))

f_bronze = f_bogus & (f_class | f_fail)
```

But note that in this case, if SIMBAD is down, only `f_bogus` will apply.